test/acceptance_tests/test_cropped_decoding.py

```diff
@@ -14,12 +14,111 @@
 
 from braindecode.datautil.iterators import CropsFromTrialsIterator
 from braindecode.models.util import to_dense_prediction_model
-from braindecode.monitors import compute_preds_per_trial_from_crops
 from braindecode.models import ShallowFBCSPNet
 from braindecode.util import set_random_seeds
 from braindecode.util import var_to_np, np_to_var
 
 
+def compute_preds_per_trial_from_crops(all_preds, input_time_length, X):
```

Comment:
this assumes a classification task right? what this code does is basically a voting classifier?
also make sure any function that is not meant to be used directly and public is preceded with a _

Reply:
This is classification task, the code sums up log probabilities per timestep for each class, so same as multiplication of probabilities per timestep for each to compute probability of trial for each class.

Reply:
Made the underscore change